asistente_ladm_col/config/general_config.py

```diff
@@ -68,6 +68,9 @@
 # For testing if an schema comes from ili2db
 INTERLIS_TEST_METADATA_TABLE_PG = 't_ili2db_table_prop'
 
+# Endpoint for testing the Source Service
+DEFAULT_ENDPOINT_SOURCE_SERVICE = "https://ide.proadmintierra.info/ws/"
```

Comment:
It would be better another name, it is not an endpoint, the endpoint is /ws, that is the complete URL of the SOURCE service?

Reply:
Any reference you can provide us with?

Reply:
I coun't find any valid and unificated definition of "endpoint" in a reference to resolve this. URI is a proposal for a more extended naming convetion, this is the correct way to name it. https://tools.ietf.org/html/rfc3986 https://prateekvjoshi.files.wordpress.com/2014/02/uri-vs-url-vs-urn.jpg

Reply:
BASE URL is an alternative if you want append more parameter to construct an URI.

Reply:
There is a large number of references that use the term `endpoint` as a URL that identifies a web service URL where you can start performing several HTTP operations from. That's a well spread (i.e., *de facto*) use for the term. Even the W3C uses it. 

Honestly, I don't think this deserves so much attention. There are parts of the plugin that truly need our focus.